pkgs/by-name/au/audit/package.nix

```diff
@@ -78,12 +75,26 @@ stdenv.mkDerivation (finalAttrs: {
     "--disable-zos-remote"
     "--with-arm"
     "--with-aarch64"
+    "--with-io_uring"
+    # allows putting audit files in /run/audit, which removes the requirement
+    # to wait for tmpfiles to set up the /var/run -> /run symlink
+    "--runstatedir=/run"
     # capability dropping, currently mostly for plugins as those get spawned as root
     # see auditd-plugins(5)
     "--with-libcap-ng=yes"
     (if enablePython then "--with-python" else "--without-python")
   ];
 
+  # remove legacy start/stop scripts to remove a bash dependency in $lib
+  # See also https://github.com/linux-audit/audit-userspace?tab=readme-ov-file#starting-and-stopping-the-daemon
+  # We could pass --disable-legacy-scripts to configure instead, but
+  # that might cause compliance issues. $bin depends on bash anyways,
+  # we have the augenrules script.
+  postInstall = ''
+    installShellCompletion --bash init.d/audit.bash_completion
+    moveToOutput libexec/libexec/initscripts/legacy-actions "$bin"
+  '';
```

Comment:
After the README updates upstream, i decided its probably best for now to keep the legacy scripts for people that are interested in compliance. I plan to remove them once i figured out a way to log who triggers systemctl commands. Systemd is already aware of who tries to do systemctl stuff, and it knows about audit. I imagine we could just add a piece of code that logs an audit entry whenever someone tries something funny. But that is a project for future me.

Reply:
On a FHS distro, they are also by default in libexec, so I don't know who uses them anyways. I agree, they should be removed and if you really care about this, you should just execute the shutdown yourself with `auditctl --signal`.